src/pages/blog/index.tsx

```diff
@@ -160,32 +166,33 @@ const BlogPage: FC<{ data: BlogGQL }> = ({ data }) => {
           <Box className={classes.blogContainer}>
             <SectionTitle className={classes.title}>Blog</SectionTitle>
             <Box className={classes.blogPosts}>
-              {blogData.blogPost.map((blogPost, index) => (
-                <div key={`${blogPost.blogTitle}-${blogPost.blogDescription}`}>
-                  <BlogPost
-                    className={classes.blogPost}
-                    image={blogPost.blogImage.publicURL}
-                    tagName={blogPost.blogLabel}
-                    text={blogPost.blogDescription}
-                    title={blogPost.blogTitle}
-                    date={dayjs(blogPost.publishDate).format('DD MMMM YYYY')}
-                    onClick={() => setSelectedBlogpost(index)}
-                  />
-                  <BlogPostDialog
-                    contentheader={blogPost.blogDescription}
-                    contentmain={blogPost.blogBody}
-                    imgurl={blogPost.blogImage.publicURL}
-                    isOpen={index === selectedBlogpost}
-                    subtitle={dayjs(blogPost.publishDate).format('DD MMMM YYYY')}
-                    tagtitle={blogPost.blogLabel}
-                    title={blogPost.blogTitle}
-                    type={componentType}
-                    handleClose={() => handleCloseBlogpost()}
-                    handlePrev={() => handlePrevBlogpost(index)}
-                    handleNext={() => handleNextBlogpost(index)}
-                  />
-                </div>
-              ))}
+              {blogData.blogPost !== undefined &&
```

Comment:
https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
Nice read, thanks, updated to ternary because I'm not a fan of `ifs` just like the author 😄